bin/src/learner/predict.py

```diff
@@ -30,7 +30,8 @@ def predict(self) -> dict:
         # get the predictions for each batch
         with torch.no_grad():
             for x, y, meta in self.dataloader:
-                current_predictions = self.model.batch(x, y, **self.loss_dict)[1]
+                # the loss_dict could be unpacked with ** and the function declaration handle it differently like **kwargs. to be decided, personally find this more clean and understable.
+                current_predictions = self.model.batch(x=x, y=y, loss_fn=self.loss_dict)[1]
```

Comment:
loss_fn should not be hardcoded, because one may give loss_fn1, loss_fn2, or anykind of keyword